words/headers.asm

```diff
@@ -41,7 +41,17 @@
 ;       AN - Always Native Compile (may not be called by JSR)
 ;       UF - Contains underflow check
 ;       HC - Has CFA (words created by CREATE and DOES> only)
-
+;
+;       The NN and AN flags are intrepreted together like this:
```

Comment:
Should add this table to the docs as well, perhaps in ch_internals.adoc?  It would be nice to indicate this (NN+AN=ST) in the disasm output, but there are so few of these words and it's unlikely the user will be writing words with both flags set, so I don't think it's worth the bytes and effort.  I think having it in the docs should be sufficient.

Reply:
I wanted to improve the flags output in `SEE` anyway, so took the opportunity to add calculated flags like UF and ST at the same time.   Now it interleaves the values with the names, and shows a calculated section following a pipe symbol.
```
see r@ 
nt: BC6A  xt: 8EB5 
flags: CO 1 IM 0 AN 1 NN 1 HC 0 | UF 0 ST 1 
size (decimal): 20 

8EB5  68 7A 1A 85 25 D0 01 C8  84 26 CA CA 7A 94 00 68  hz..%... .&..z..h
8EC5  95 01 48 5A  ..HZ

...

see drop 
nt: BC15  xt: 8660 
flags: CO 0 IM 0 AN 0 NN 0 HC 0 | UF 1 ST 0 
size (decimal): 5 

8660  20 89 D8 E8 E8   ....

8660   D889 jsr     1 STACK DEPTH CHECK
...
```